### PR TITLE
cypress test all script

### PR DESCRIPTION
**Pull request by tonypls**
## Issue
As part of https://linear.app/electricitymaps/issue/ELE-1832/automate-deployment-to-staging

## Description
Adds a script to package.json to run both cypress test at once

### CODE REVIEW

**Comment by tonypls** [comment]
Please review me <3

File: web/package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "prebuild": "pnpm run create-generated-files",
     "cy:component": "cypress open --component -b electron",
     "cy:e2e": "cypress open --e2e -b electron",
+    "cy:test-all": "cypress run && cypress run --component",
```
Comment (VIKTORVAV99) [inline web/package.json:25]: should this also use the electron browser as the cy:component script or does that not matter?

**Review by VIKTORVAV99 — APPROVED** [review]
Looking good but I have one question.

File: web/package.json (hunk 1 above)
Comment (tonypls) [reply web/package.json:25]: Electron is the default :)

**Pull request merged**